src/app/modules/form/form.errors.ts

```diff
@@ -51,3 +51,12 @@ export class TransferOwnershipError extends ApplicationError {
     super(message)
   }
 }
+
+/**
+ * Error to be returned when form logic cannot be found
+ */
+export class LogicNotFoundError extends ApplicationError {
+  constructor(message: string) {
```

Comment:
nit: should we have a default message for this error? Then instantiating this error won't require an argument anymore

Reply:
edited